### PR TITLE
Add homebrew documentation to readme

### PR DESCRIPTION
**Pull request by tomasharkema**
as of https://github.com/mac-cain13/R.swift/issues/60, r.swift is available in the homebrew package manager.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -82,13 +82,32 @@ _Tip:_ Add the `*.generated.swift` pattern to your `.gitignore` file to prevent
 ### Manually
 
 0. Add the [R.swift.Library](https://github.com/mac-cain13/R.swift.Library#Installation) to your project
-1. [Download](https://github.com/mac-cain13/R.swift/releases) a R.swift release, unzip it and put it into your source root directory
+1. [Download](https://github.com/mac-cain13/R.swift/releases) a R.swift release, unzip it and put it into your source root directory OR install the binary via homebrew. For more detail check [Homebrew](#homebrew)
```
Comment (mac-cain13) [inline Readme.md]: The installation instruction in step 3 is using `"$SRCROOT/rswift"`, that won't work when you use Homebrew.

**Comment by mac-cain13** [comment]
Thanks for the PR and making R.swift available on Homebrew! :)

The manual installation edit will not work using homebrew, because of the path, I would suggest to make the `### Homebrew` section just like `Cocoapods` and `Manual` and put a step by step instruction. The path the the `rswift` binary is different.

Also the whole output of the `rswift -h` call is something I would take out, just saying that you can call `rswift -h` is enough I think. (We'll definitely will forget to updated the readme when the help output changes...)

**Comment by tomasharkema** [comment]
Maybe some input for a discussion: should we replace the manual installment via homebrew? or

**Comment by mac-cain13** [comment]
Ping @tomasharkema.

Are you going to give this PR an update?

**Pull request merged**